<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>图片识别</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="0">
    <widget class="QComboBox" name="cmbmax">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>默认</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500*500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600*600</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="1" rowspan="2">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>缩放倍率:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="slider">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>调整模型参数</string>
      </property>
      <property name="minimum">
       <number>101</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="sliderPosition">
       <number>110</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pbtRun">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>识别</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QComboBox" name="cmbmin">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>默认</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50*50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100*100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200*200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>模型选择:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="cmbmodel">
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>人脸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>眼睛</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>左眼</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>全身</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_7">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>MAX:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="radauto">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>窗口自适应</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MIN:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>识别次数：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radwh">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>固定长宽比</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>目标范围：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pbtBack">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbtSelect">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="cmbtimes">
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" rowspan="11">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
